tequilapi/endpoints/identities.go

```diff
@@ -32,37 +32,18 @@ import (
 )
 
 type balanceGetter func(id identity.Identity) uint64
-
 type balanceFetcher func(id string) (pingpong.ConsumerData, error)
+type earningsGetter func(id identity.Identity) uint64
```

Comment:
Such function types make it hard to navigate to the relevant implementation. I have to go to the constructor, find the matching parameter, then look for the usages of the constructor and determine matching argument (which is usually positional). Sometimes there are more layers, thus multiplying the time needed to navigate to the implementation.

Were it written as:
```go
type earningsGetter interface {
	Balance(id identity.Identity) uint64
}
```
You could use ⌘+⌥+B or ⌘+Click to navigate directly to implementation(s). For easier navigation, I suggest using an interface with a single method.
In case you're wondering, I'll also propose it to the devbook.

Reply:
I known, i wrote this before communication. Refactored.